csss-site/src/events/templates/events/fall_hacks_submissions2020.html

```diff
@@ -0,0 +1,329 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>
+
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+
+  </head>
+  <body>
+    <section id='above-fold'>
+      <header>
+        <h1 class="title">Project Page of submissions from the Computing Science Student Society Fall Hacks 2020</h1>

```

Comment:
maybe update that string to `Computing Science Student Society Fall Hacks 2020 Submissions`?